app.js

```diff
@@ -1,21 +1,19 @@
-var BrowserWindow = require('browser-window')
-var path = require('path')
+const { BrowserWindow, app } = require('electron')
+const path = require('path')
 
-var APP_NAME = 'Whiteboard'
-var INDEX = 'file://' + path.join(__dirname, 'static/index.html')
+const APP_NAME = 'Whiteboard'
+const INDEX = 'file://' + path.join(__dirname, 'static/index.html')
 
-var app = require('app')
 app.on('ready', appReady)
 
-var mainWindow
```

Comment:
You need to move the variable to the top level or else the local variable in `appReady` may get garbage-collected and then the window will get closed.